stellarphot/gui_tools/seeing_profile_functions.py

```diff
@@ -392,9 +392,9 @@ def show_event(
             else:
                 # User changed aperture
                 aperture_radius = aperture["radius"]
-                ap_settings = ApertureSettings(
+                ap_settings = PhotometryApertures(
                     **aperture
-                )  # Make an ApertureSettings object
+                )  # Make an aperture settings object, but don't update it's widget yet.
```

Comment:
Maybe update this to be clear its an aperture photometry settings related thing.